lib/blackjack_score.rb

```diff
@@ -1,24 +1,51 @@
 class BlackjackScore
 
   def get_hand_value(hand)
-    get_card_value(hand[0])
-  end
+    card_values = []
+    hand.each do |card|
```

Comment:
There is a pretty handy (😄) method on `hand` called `map` which maps each `card` to the result of any function you want and return the finished array. 
https://ruby-doc.org/core-2.4.1/Enumerable.html#method-i-map